components/x11/xf86-input-vmmouse/manifests/sample-manifest.p5m

```diff
@@ -25,10 +25,7 @@ license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 file path=etc/hal/fdi/policy/10osvendor/11-x11-vmmouse.fdi
 file path=usr/lib/$(MACH64)/xorg/modules/input/vmmouse_drv.so
 file path=usr/lib/hal/$(MACH64)/hal-probe-vmmouse
-file path=usr/lib/hal/hal-probe-vmmouse
 file path=usr/lib/xorg/$(MACH64)/vmmouse_detect
-file path=usr/lib/xorg/modules/input/vmmouse_drv.so
-file path=usr/lib/xorg/vmmouse_detect
```

Comment:
should hal call these handlers? If so, do we still need 32-bit ones?

Reply:
Probably we need to adjust the path in hal.

Reply:
Perhaps, it'll be ok with 64-bit binaries in 32-bit path.

Reply:
but someone should look at hal-probe-vmmouse usage and test it :)

Reply:
/usr/lib/hal/amd64/hal-probe-vmmouse contains the full path to /usr/lib/xorg/amd64/vmmouse_detect so no problem, probably I should just move /usr/lib/hal/amd64/hal-probe-vmmouse to /usr/lib/hal/hal-probe-vmmouse

Reply:
Done